en/docs/choreo-concepts/choreo-marketplace.md

```diff
@@ -0,0 +1,59 @@
+# Choreo Marketplace
+
+Choreo Marketplace provides a comprehensive collection of prebuilt connectors and triggers to design your cloud native applications. It also serves as a platform for API developers to publish their own connectors for Choreo users, either to purchase or to use free of charge.
+
+The Choreo Marketplace promotes API reuse. This empowers modern businesses to focus on developing applications with unique functionality while saving resources that otherwise would be required to create everything from scratch.
+
+When you discover a connector or trigger in the Choreo Marketplace that meets your requirements, you have the flexibility to choose between purchasing it, acquiring it through a Freemium account, or using it for free based on its pricing.
+
+
+## Access the Choreo Marketplace
+
+To access the Choreo Marketplace, follow the steps given below:
+
+1. Sign in to the [Choreo Console](https://console.choreo.dev/).
+
+2. In the left navigation menu, click **Marketplace**. This opens the Choreo Marketplace with the prebuilt connectors designed and published by Choreo. 
```

Comment:
Since Choreo is a platform, it is wrong to say designed and published by Choreo. Either you can say, Designed and Published by WSO2 employees. But here I think it's irrelevant as to who published it. Shall we just say, prebuilt connectors?

Reply:
Done.